SAST/AzureDevOps/azure-pipelines_sast.yml

```diff
@@ -0,0 +1,18 @@
+trigger:
+- main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- script: curl -LJO https://downloads-demo.whitesourcesoftware.com/sast-cli/linux/wscli && chmod +x wscli
+  displayName: 'Downloading WS SAST Tool'
+
+- script: ./wscli --dir ./ --name=AD_$(System.TeamProject) -app AZDO_Pipeline
```

Comment:
name and app should be the following
-name $(Build.Repository.Name)_$(Build.SourceBranchName)
-app AZ_$(System.TeamProject)

Reply:
Fixed as Shai is away.